src/Slider.jsx

```diff
@@ -1,18 +1,14 @@
 /* globals document */
 
 import React, { PropTypes } from 'react';
+import { css, withStyles } from 'react-with-styles';
+
+import DefaultHandle from './components/DefaultHandle';
+import DefaultProgressBar from './components/DefaultProgressBar';
 
 import * as SliderConstants from './constants/SliderConstants';
```

Comment:
I know you didn't do this, but I'm not a fan of `import *`. What do you think about making this explicit?